File: 1주차/4344.py
n = int(input())

for i in range(n):
    
    arr = list(map(int,input().split()))
    count = 0
    sum1 = sum(arr[1:])
    avg = sum1/arr[0]
    
    for j in arr[1:]:
        if avg < j:
            count += 100

    result = count/arr[0]   
    print("%0.3f%%" % result)
        

File: 1주차/4344.c
#include <stdio.h>

int main()
{

	int i, j,z, n,m;
	int arr[20][20] = { 0 };
	int sum[20] = { 0 };
	int count[20] = { 0 };
	double result[20] = { 0.0 };

	scanf_s("%d", &n);

	for (i = 0; i < n; i++)
	{
		scanf_s("%d", &m);
		for (j = 0; j < m; j++)
		{
			scanf_s("%d", &arr[i][j]);
			sum[i] += arr[i][j];
		}
		result[i] = sum[i] / m;
		

		for (z = 0; z < m; z++)
		{
			if (arr[i][z]>result[i])
				count[i] += 100;
			//printf("count: %d \n", count[i]);
		
		}
		result[i] =  (double)count[i] / (double)m;

	}

	for (i = 0; i < n; i++)
		printf("%.3f%%\n", result[i]);
		

}